packages/app/src/containers/Canvas.tsx

```diff
@@ -98,6 +99,11 @@ export default memo(function Canvas() {
   const containerSize = useSize(containerRef);
   const meta = useSelector(Selectors.getCurrentPageMetadata);
   const { setCanvasSize, highlightLayer, highlightedLayer } = useWorkspace();
+  const bind = useGesture({
+    onWheel: ({ delta: [x, y] }) => {
+      dispatch('gesturePan', { x, y });
```

Comment:
Should we be more specific and call the action `wheelPan`?

Reply:
Or maybe more general and call it `pan`. We already have `panning`, `maybePan`, `panMode`, etc, without any qualifiers, so I think that'd be more consistent. Maybe at some point I'll update those to use this new `pan`